src/Components/Onboarding/Views/OnboardingWelcome.tsx

```diff
@@ -61,6 +61,7 @@ export const OnboardingWelcome = () => {
               as={RouterLink}
               to="/"
               onClick={onClose}
+              data-test="onboarding-skip-button"
```

Comment:
(totally no blocking trivial comment): worth noting that in a JS world there's no need to use hyphens in variable values and they're pretty foreign syntax-wise (and mostly a legacy html idiom). `data-test="onboardingSkipButton"` works just as well, matches rest of the code

Reply:
hey, fixing the integration tests now so ill update on a follow-up pr addressing new integration tests for onboarding flow.

Reply:
no need to follow up! just a side comment for the future